### PR TITLE
Document namespace prefix when scanning a hosted cluster

### PR DESCRIPTION
**Pull request by yuumasato**
When using hypershift and scanning a hosted cluster, the namespace of the hosted cluster needs to be defined as well.

The namespace where clusters are installed can vary, and should be set accordingly for th scans to work.

### CODE REVIEW

File: doc/usage.md [hunk 1]
```diff
@@ -576,11 +576,15 @@ error result.
 Compliance Operator is able to run a platform scan on the HyperShift Managment cluster
 for the Hosted Cluster with a tailoredProfile.
```
Comment (rhmdnd) [inline doc/usage.md:577]: A picture describing, or better, linking to the architecture of hypershift would be useful.

Reading this from the perspective of someone who doesn't know what hypershift is, it's hard to discern what "hosted" versus "management" clusters are.

We can link to https://hypershift-docs.netlify.app/reference/concepts-and-personas/ to describe those concepts and reuse them here.

Comment (yuumasato) [reply doc/usage.md:578]: Agree, what do you think about https://github.com/ComplianceAsCode/compliance-operator/pull/382/commits/90fbb120ea0d05960485e4ac9c3c1574a20ad823 ?

File: doc/usage.md [hunk 2]
```diff
@@ -573,14 +573,22 @@ error result.
 
 ## How to Use Compliance Operator with HyperShift Management Cluster
 
-Compliance Operator is able to run a platform scan on the HyperShift Managment cluster
+Hypershift[2] allows one to create and manage clusters on existing infrastructure.
+Compliance Operator is able to create a platform scan on the HyperShift Management cluster[3]
 for the Hosted Cluster with a tailoredProfile.
```
Comment (rhmdnd) [inline doc/usage.md:578]: nit: `TailoredProfile`

File: doc/usage.md [hunk 3]
```diff
@@ -573,14 +573,22 @@ error result.
...
-Currently, we only support CIS profile and PCI-DSS profile, in order to scan Hosted
-Cluster, you need to create a tailoredProfile and then set the value of
-`ocp4-hypershift-cluster` to the name of the Hosted Cluster you want to scan,
+[2]: https://hypershift-docs.netlify.app/
+[3]: https://hypershift-docs.netlify.app/reference/concepts-and-personas/
+
+Currently, we only support CIS profile and PCI-DSS profile,
 you can either extend `ocp4-cis` or `ocp4-pci-dss`.
 
+In order to scan a Hosted Cluster, you need to create a tailoredProfile to define the
```
Comment (rhmdnd) [inline doc/usage.md:586]: nit: `TailoredProfile`

**Review by rhmdnd — COMMENTED** [review]
Looks good, I only have minor comments inline about style. Thank you for adding the reference to the Hypershift documentation.

File: doc/usage.md [hunk 4]
```diff
@@ -620,7 +631,7 @@ settingsRef:
 
 ## How to Use Compliance Operator with HyperShift Hosted Cluster
 
-Compliance Operator is able to run a platform scan on the HyperShift Hosted cluster
+Compliance Operator is able to run a platform scan on the HyperShift Hosted cluster[3]
```
[the remark this answers is not in the document]
Comment (yuumasato) [reply doc/usage.md:634]: The numbered links are not rendered in the document. And the numbers are actually clickable.

But I think the inline links are nicer to read and more evident.

**Comment by yuumasato** [comment]
I have fixed a small typo and rephrased one sentence.
Changed from `define the
name and namespace` to  `specifying the name and namespace`

File: doc/usage.md [hunk 5]
```diff
@@ -573,14 +573,19 @@ error result.
...
+for the Hosted Cluster with a TailoredProfile.
 
-Currently, we only support CIS profile and PCI-DSS profile, in order to scan Hosted
-Cluster, you need to create a tailoredProfile and then set the value of
-`ocp4-hypershift-cluster` to the name of the Hosted Cluster you want to scan,
+Currently, we only support CIS profile and PCI-DSS profile,
 you can either extend `ocp4-cis` or `ocp4-pci-dss`.
 
+In order to scan a Hosted Cluster, you need to create a TailoredProfile specifying the
+name and namespace of the Hosted cluster that you want to scan.
```
Comment (rhmdnd) [inline doc/usage.md:584]: nit: Hosted Cluster (to be consistent with the line above).

File: doc/usage.md [hunk 6]
```diff
@@ -573,14 +573,19 @@ error result.
 
 ## How to Use Compliance Operator with HyperShift Management Cluster
 
-Compliance Operator is able to run a platform scan on the HyperShift Managment cluster
-for the Hosted Cluster with a tailoredProfile.
+[Hypershift](https://hypershift-docs.netlify.app/) allows one to create and manage clusters on existing infrastructure.
+Compliance Operator is able to create a platform scan on the [HyperShift Management Cluster](https://hypershift-docs.netlify.app/reference/concepts-and-personas/)
+for the Hosted Cluster with a TailoredProfile.
```
Comment (rhmdnd) [inline doc/usage.md:578]: Similar to my comments on another review. What are our thoughts on using literals for formal CRD names?

**Review by rhmdnd — COMMENTED** [review]
Looks good, only two minor questions.

File: doc/usage.md (hunk 5 above)
Comment (yuumasato) [reply doc/usage.md]: Addressed.

File: doc/usage.md (hunk 6 above)
Comment (yuumasato) [reply doc/usage.md]: Should be fixed as well.

**Review by rhmdnd — APPROVED** [review]
/lgtm

**Comment by rhmdnd** [comment]
/px-approved
/doc-approved
/qe-approved

Applying necessary labels since this documenting existing behavior. Adding @sheriff-rh for downstream documentation awareness.

**Pull request merged**